isyfact-standards-doc/src/docs/antora/modules/blaupausen/pages/referenzarchitektur/inhalt.adoc

```diff
@@ -468,6 +468,45 @@ Für derartige Aufrufe gibt es keine Vorgaben: Sie können aus beliebigen Schich
 [id="image-CallFAmGuX",reftext="{figure-caption} {counter:figures}"]
 image::blaupausen:referenzarchitektur/CallFAmGuX.png[align="center"]
 
+[[zentrale-sicherheitsaspekte]]
+=== Zentrale Sicherheitsaspekte in der Referenzarchitektur
+
+Die IsyFact Referenzarchitektur legt fest, dass jegliche Nutzung von IsyFact-Anwendungen nur authentifizierten und dafür autorisierten Benutzern möglich ist.
+
+Von Außen können IsyFact-Anwendungen und deren Services nur über Zugangsservices in der Informations- u. Dienstezone erreicht werden:
+
+- Interne und externe menschliche Anwender, die die Anwendung über deren Web-Oberfläche (im Web-Browser) bedienen, werden über das Portal geleitet.
+
+- Externe Systeme werden über ein Service-Gateway geleitet.
+In zukünftigen Versionen von IsyFact muss das externe System erst über das Authentication-Gateway die Berechtigung zur Nutzung einholen.
+
+Interne Systeme, die auf Services anderer IsyFact-Anwendungen in der selben Anwendungslandschaft zugreifen wollen, müssen auch erst die Berechtigung zur Nutzung einholen.
+
+.Integration von IAM in eine Anwendungslandschaft
+[id="image-iam-integration-awl",reftext="{figure-caption} {counter:figures}"]
+image::referenzarchitektur/sicherheit_iam_integration_awl.dn.svg[]
+
+Die Zugangsberechtigung wird dabei in zwei Schritten überprüft:
+
+. Die Zugangsservices prüfen die Identität des Aufrufers und lassen nur für die Anwendungslandschaft zugelassene durch.
+Für die Authentifizierung wird ein zentraler Service (IAM-Service) in der Anwendungslandschaft genutzt.
+Der IAM-Service verifiziert die Identität des Anwenders bzw. Systems anhand eindeutiger Eigenschaften (z.B. biometrische Daten) oder geheimer Daten (z.B. Passwort).
+Die Identität und die damit verknüpften Rollen in der Anwendungslandschaft werden in der Anwenderverwaltung gepflegt.
```

Comment:
Wenn man dies erwähnt, dann ist man auf einer technischen Ebene, in der der Leser dann wirklich auch die Details wissen will: was passiert mit den gepflegten Berechtigungen? Wo werden sie übermittelt, wie werden sie ausgewertet?
und wie wird sichergestellt dass diese Informationen nicht verfälscht werden